inventory/docker-it/src/test/java/org/opennms/horizon/inventory/cucumber/steps/InventoryProcessingStepDefinitions.java

```diff
@@ -493,11 +492,11 @@ public void deviceTaskIPAddress(String ipAddress) {
     public void monitorType(String monitorType) {
```

Comment:
This can be short-cutted:
```suggestion
    public void monitorType(String monitorType) { this.monitorType = monitorType; }
```